packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -690,28 +671,4 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
             1e18 // Canonical value representing "True"; i.e. the proposed relay is valid.
         );
     }
-
-    function _emitDepositRelayedEvent(
```

Comment:
+1 I only added this to beat the stack too deep compile issue, which seems to have disappeared

Reply:
Yeah, the struct helped a lot with that.